docs/microservices.json

```diff
@@ -68,12 +73,17 @@
                     "apis": []
                 },
                 {
-                    "id": "idam-web",
+                    "id": "authentication-web",
+                    "hard": true,
+                    "apis": []
+                },
```

Comment:
does CCD API Gateway talk to idam frontend?

Reply:
I don't think so. Their front end is an angular app which talks to the API gateway

Reply:
so this should be removed then

Reply:
I'd need @vlaurin to confirm but iirc this gateway does some IdAM user verification for them?

Reply:
yeah, maybe `idam-app`. definitely not the web frontend.

Reply:
Yes, CCD's gateway talks to `idam-app` and `service-auth-provider` but not to the IDAM front-ends